scripts/listapranzo.js

```diff
@@ -85,6 +85,29 @@ module.exports = function (robot) {
     order.users[user.id] = dishes;
   };
 
+  var fuzzyMatch = function (dish, menuline) {
+    dish = dish.trim().toLowerCase().replace(" ", "");
+    var key = "";
+    for (var i = 0; i < dish.length; i++)
+      key = key + dish[i] + ".*"
```

Comment:
Beh se uno sa che è univoco non vedo il problema.. e se non lo è in ogni caso ti vengono ritornati i vari match chiedendoti quale dei vari è quello giusto. Fare fuzzy finding è un supercase di substring matching, quindi nessuno ti vieta di scrivere arrabb se vuoi. Non vedo perchè levare l'opportunità di scrivere NB se a qualcuno va di farlo.

Capisco anche se dici che con fuzzy un caso che era facile di substring potrebbe fallire perchè essendo fuzzy matcha più cose.. ma su entry di generalmente tre parole qualè la possibilità? Se scrivi arrabb ci dovrebbe essere anche una cosa tipo "Aragosta al rabarbaro blu" per dirti che ci sono più match. Ma anche lì secondo me il vantaggio di poter scrivere "paarrab" per pasta all'arrabbiata è maggiore che la possibilità di aumentare le collisioni (e tieni conto che se ci sono collisioni ti dice i match, quindi non ti ordina una cosa a caso, e ti prende per quell'unica volta su 100 un altro mezzo secondo per ripremere - contro eventuali typo che puoi sbagliare con substring).